### PR TITLE
[Readme] Add instructions for multiple applications on same domain

### PR DESCRIPTION
**Pull request by BlackbitDevs**
Follow-up of https://github.com/pimcore/skeleton/pull/198#issuecomment-2395958966

### CODE REVIEW

File: doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+When trying to log in at http://example.org/admin you will get a 403 Forbidden error.
+(The reason why Magento sets the cookie for `.example.org` instead of `example.org` probably is to also support subdomains.)
+
+You can prevent this problem by setting
+```yaml
+framework:
+    session:
+        name: "PIMCORE_SESSION_ID"
+```
+in your `config.yaml`. This way the session cookies do not conflict anymore and you will be able to log in to both applications at the same time.
```
Comment (mattamon) [inline doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md]: I would move this up before the code block for a better reading flow.

File: doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+When running multiple applications on the same domain, there can be session cookie collisions which prevent you to log in to both systems at the same time.
+
+Imagine you run a Magento app on http://example.org and Pimcore on http://pim.example.org. Then you will have 2 cookies with name `PHPSESSID` (if `session.name` in php.ini is the same for both):
+| Name| Value  | Domain  |  Path | 
+|---|---|---|---|
+| PHPSESSID | 5a9b08750387d9e11c738a2947d93e38   |  .example.org | /  | 
+| PHPSESSID | irqnjh5p96gp2i8iu743ulm32p   |  pim.example.org | /  | 
+
+First one is from Magento, second from Pimcore.
```
Comment (mattamon) [inline doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md]: Not sure If I like that we mention Magento in our docs. @fashxp WDYT?

**Review by mattamon — CHANGES_REQUESTED** [review]
Rest looks good as is! Thank you for updating the docs!

File: doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md (hunk 2 above)
Comment (BlackbitDevs) [reply doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md]: Changed to "web shop"

File: doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md (hunk 1 above)
Comment (BlackbitDevs) [reply doc/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/08_Multi_Application_Setup.md]: Done

**Review by mattamon — APPROVED** [review]
LGTM. Thank you for the follow up!

**Pull request merged**